src/Decisions/index.js

```diff
@@ -1,8 +1,16 @@
 import React from 'react'
-import Grid from './Grid'
-import GridContent from './Grid/GridContent'
+import _ from 'lodash'
+import DisputesTable from '../DisputesTable'
 import './Decisions.css'
 
+const filterFunction = disputes => {
+  return _.filter(disputes, dispute => {
+    if ((dispute.disputeData.isJuror && dispute.disputeData.hasRuled) || !dispute.disputeData.isJuror) {
+      return dispute
+    }
+  })
+}
```

Comment:
`const filterFunction = disputes =>  _.filter(disputes, dispute => ((dispute.disputeData.isJuror && dispute.disputeData.hasRuled) || !dispute.disputeData.isJuror) && dispute)` 
I don't know if it's works but if you can use the functional syntax it's better